dict/P/pandunia.tsv

```diff
@@ -455,6 +457,8 @@ PWN:committee.n.01		komisia	ko.mis·ia
 PWN:common.a.01		konhave	kon·hav'e	
 PWN:companion.n.01		sahib	sahib	ara: صَاحِب (ṣāḥib), fas: صاحب (sāhib), hin:साहिब (sāhib), ben:সাহেব (śaheb), may:sahabat, swa:sahibu
 PWN:compare.v.03		kompar	kom.par	
+PWN:compass.n.01		busola	busola	fra:boussole, spa:brújula, por:bússola, ara:بُوصُلَة (būṣula), tur:pusula, rus:буссо́ль (bussól’)

```

Comment:
This word sounds nice but I think that a compound word would be fine, like Chinese _north-pointing-needle_ or Persian _pole-pointer_. So how about _gik pin_ ('pole needle') or _gik dikater_ ('pole pointer')?

Reply:
That makes sense.  But shouldn't it be _gik dikator_ instead of _gik dikater_ since it's a tool and not a person?